app/assets/stylesheets/landing_page/sections/hero.scss

```diff
@@ -166,6 +167,10 @@
   color: #FFFFFF;
   letter-spacing: 0px;
 
+  outline: none;
```

Comment:
Focus outline is an important accessibility feature when tabbing through links, buttons, etc. to see which one is focused. You should make sure the button has some visual styles that differentiate when it is in focus.